plugins/groq/package.json

```diff
@@ -57,9 +58,9 @@
   "scripts": {
     "check": "tsc",
     "compile": "tsup-node",
-    "build:clean": "rm -rf ./lib",
+    "build:clean": "rimraf ./lib",
     "build": "npm-run-all build:clean check compile",
     "build:watch": "tsup-node --watch",
-    "test": "node --import tsx --test ./tests/*_test.ts"
+    "test": "tsx \"./tests/groq_test.ts\""
```

Comment:
Can we keep the test script generic? Now it just runs the one test file

Reply:
I was having problems running it in my PC. That is why I did that I will try to do a generic one.

Reply:
I have made a generic version. I will commit the change.